pages.ta/windows/choco-install.md

```diff
@@ -0,0 +1,36 @@
+# choco install
+
+> சாக்லேட்டியுடன் ஒன்று அல்லது அதற்கு மேற்பட்ட தொகுப்புகளை நிறுவவும்.
+> மேலும் தகவல்: <https://chocolatey.org/docs/commands-install>.
+
+- ஒன்று அல்லது அதற்கு மேற்பட்ட இடம் பிரிக்கப்பட்ட தொகுப்புகளை நிறுவவும்:
+
+`choco install {{package(s)}}`
+
+- தனிப்பயன் உள்ளமைவு கோப்பிலிருந்து தொகுப்புகளை நிறுவவும்:
+
+`choco install {{path/to/packages.config}}`
+
+- ஒரு குறிப்பிட்ட nuspec அல்லது nupkg கோப்பை நிறுவவும்:
```

Comment:
```suggestion
- ஒரு குறிப்பிட்ட `nuspec` அல்லது `nupkg` கோப்பை நிறுவவும்:
```
Even though this isn't present on the English page, I prefer this in translations.